media-gfx/sane_backends/sane_backends-1.0.27.recipe

```diff
@@ -59,6 +58,7 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
```

Comment:
shouldn' be needed.

Reply:
Actually, PNG/JPG/TIFF is used in most scanning software.